### ISSUE TITLE
Change locked Sodium/Iris compat version to latest 

### ISSUE THREAD

**Issue by AlienXtream** [qouteall/ImmersivePortalsMod#1482]
### Is it a mod compatibility issue?

Yes

### If it's a mod compatibility issue, please provide the link(s) to the incompatible mod(s)

https://modrinth.com/mod/iris

### Minecraft version

1.20.1

### Describe the issue

iris updated in the recently and the version check needs updating. should probably also be a warning, not a strict compatability check, so minor version updated dont prevent loading the game and the user is left to decide if there is an issue or not

**Comment by qouteall** [comment]
This mod lock Iris version because this mod uses Iris internal that may break in new versions of Iris (this happened many times at past). 

For now, you can use dependency override https://fabricmc.net/wiki/tutorial:dependency_overrides to get rid of the restriction (but it may crash in-game) 

Put `fabric_loader_dependencies.json` into `config` folder with these content:
```
{
    "version": 1,
    "overrides": {
      "immersive_portals": {
        "-breaks": {
          "iris": "IGNORED",
          "sodium": "IGNORED"
        }
      }
    }
  }
```

**Comment by qouteall** [comment]
Probably fixed in 3.3.7
